app/uk/gov/hmrc/eoricommoncomponent/frontend/views/check_your_details_register.scala.html

```diff
@@ -160,7 +161,7 @@
 @orgName = @{
     subscription.nameOrganisationDetails match {
         case Some(nameOrgDetails) => nameOrgDetails.name
-        case _ =>  registration.name
```

Comment:
Just a question: do we store a different name here?


btw good spot

Reply:
Nope, here the value is always the default 'orgName' and is never really being used. The only reason the tests were passing is that they were also using 'orgName' Thanks! 👍